pg-auto-failover.spec

```diff
@@ -82,6 +82,9 @@ PATH=%{pginstdir}/bin:$PATH
 
 
 %changelog
+* Thu Feb 11 2021 - Gurkan Indibay <gindibay@microsoft.com> 1.4.2-1
+- Official 1.4.2 release of pg_auto_failover
```

Comment:
If you decide to add a changelog item, please place the same entry here as well.